lib/ansible/modules/network/nxos/nxos_file_copy.py

```diff
@@ -100,7 +98,7 @@
     HAS_SCP = False
 
 
-def execute_show_command(command, module, command_type='cli_show'):
+def execute_show_command(command, module):
```

Comment:
But regardless of the transport, the only thing it does is `run_commands(module, [command])`. why not replace all `execute_show_command()` with that?